source/screens/caseScreens/CaseOverview.js

```diff
@@ -72,12 +74,18 @@ const computeCaseCardComponent = (caseData, form, caseType, navigation) => {
     ['03', '02'].includes(decision.typecode)
   );
 
+  const casePersonData = persons.find((person) => person.personalNumber === personalNumber);
+
   const statusType = caseData?.status?.type || '';
   const isNotStarted = statusType.includes('notStarted');
   const isOngoing = statusType.includes('ongoing');
   const isCompletionRequired = statusType.includes('completionRequired');
   const isSigned = statusType.includes('signed');
   const isClosed = statusType.includes('closed');
+  const isCoApplicant = casePersonData?.role === 'coapplicant';
+
+  const shouldShowCTAButton =
+    !isCoApplicant && (isOngoing || isNotStarted || isCompletionRequired || isSigned);
```

Comment:
This code is easy to read and very clear (y)
A thought would be to move this to a separate function in order to keep the component a bit more clean :)

Reply:
> A thought would be to move this to a separate function in order to keep the component a bit more clean :)

Agree. There is very similar code in both `CaseOverview` and `CaseSummary` for the card component, so ideally I would like to move everything to a separate component entirely, however right now that feels like out of scope for this PR maybe?

Right now the "card component" is not a typical component, instead it is computed dynamically and stored as data _within_ the case data object during an `useEffect`. I'm not sure what the idea behind doing that instead of just using it within the jsx body is, so I haven't refactored it.

Reply:
I see, well then i agree with you. Ie it's better to refactor both of these components, instead of making a quick fix :)

If my memory is correct the computed card function was introduced in order to render different cards. I strongly believe that this can be made in a better and more cleaner fashion with components and good property management :)

Reply:
We should add this techincal debt to the backlog so that we keep an record of things that we find and want to improve. If they are to complex or large to fix within a PR. Otherwise i think it's a good practice to clean up/refactor code in prs if it's straight forward and simple(y)

We might want to introduce time slots in sprints for fixing technical debt :)